packages/arch/packages/theme/src/index.js

```diff
@@ -33,6 +37,10 @@ export const globalStyles = {
     MozFontFeatureSettings: "'liga' on",
     MozOsxFontSmoothing: 'grayscale',
     WebkitFontSmoothing: 'antialiased',
+    '@media print': {
+      backgroundColor: 'white',
+      fontFamily: printFontFamily,
```

Comment:
```suggestion
      fontFamily: 'unset',
```

Reply:
`unset` will set it back to a serif font. I don't think thats better.

![image](https://user-images.githubusercontent.com/814227/76174072-1416a200-61f9-11ea-84c5-b0e250ab7a90.png)